components/contact/show_inquiry_modal.coffee

```diff
@@ -2,11 +2,19 @@ _ = require 'underscore'
 Backbone = require 'backbone'
 ContactView = require './view.coffee'
 analyticsHooks = require '../../lib/analytics_hooks.coffee'
+openInquiryQuestionnaireFor = require '../inquiry_questionnaire/index.coffee'
+User = require '../../models/user.coffee'
 { modelNameAndIdToLabel } = require '../../analytics/helpers.js'
 Partner = require '../../models/partner.coffee'
 { SESSION_ID, API_URL } = require('sharify').data
+
 formTemplate = -> require('./templates/inquiry_show_form.jade') arguments...
 headerTemplate = -> require('./templates/inquiry_partner_header.jade') arguments...
+
+class ProxyArtwork extends Backbone.Model
```

Comment:
Kind of cheating here, we want to use the inquiry questionnaire component to finish the flow (if the user lands in the correct bucket). That component expects an artwork, in this case _only_ to render the type of partner that the inquiry is being sent to.